layers/+lang/flow-type/README.org

```diff
@@ -0,0 +1,38 @@
+#+TITLE: flow-type layer
+
+* Description
+This layer adds some [[https://flowtype.org/][flow]] related functionality to Emacs:
+ - Show the flow-type of the object under the cursor using Eldoc
+ - Add flow checking to flycheck (based on the flycheck-flow package)
+ - company-mode completion using company-flow
+ - "compilation" using =flow status= and =flow check=
+ - auto staring =flow server= in a project specific comint buffer
+
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =flow-type= to the existing =dotspacemacs-configuration-layers= list in this
+file. Please ensure you already have the flow binary installed and in your PATH.
+
+To prevent the flow server from auto-starting, set =flow-type-no-auto-start= to =t=.
+To start the server in a comint buffer on entering a =js2-mode= buffer, set it to ='process=.
+
+#+BEGIN_SRC emacs-lisp
+(flow-type :variables flow-type-no-auto-start 'process)
```

Comment:
I think it's better to phrase it as positive for presumably better ease of use, but I can totally understand the ease of implementation argument, gone that way many times myself. Just a random suggestion, shouldn't be a showstopper either way.